src/pages/CropSelector/CropCalendarView/RenderCrops.js

```diff
@@ -50,7 +50,7 @@ const RenderCrops = ({ setModalOpen, modalOpen, setModalData }) => {
         key={`cropRow${index}`}
         style={hasGoalRatingTwoOrLess(selectedGoalsRedux, crop) ? { opacity: '0.3' } : {}}
       >
-        <TableCell sx={{ padding: 0 }}>
+        <TableCell sx={{ padding: 0, backgroundColor: selectedCropIdsRedux.includes(crop.id) && '#EAEAEA' }}>
```

Comment:
Is it possible to set backgroundColor to the `<TableRow/>` instead setting it in every `<TableCell/>`?